vnext/Microsoft.ReactNative.Cxx/JSValue.natvis

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<AutoVisualizer xmlns="http://schemas.microsoft.com/vstudio/debugger/natvis/2010">
+  <Type Name="winrt::Microsoft::ReactNative::JSValue">
+    <DisplayString Condition="m_type == JSValueType::Null">null</DisplayString>
+    <DisplayString Condition="m_type == JSValueType::Object">{m_object}</DisplayString>
+    <DisplayString Condition="m_type == JSValueType::Array">{m_array}</DisplayString>
```

Comment:
Should we specialize even further that if m_object is a function, to indicate this as well?